CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Breaking changes: Add `KarlPaymentButtonVirement`, `KarlPaymentButtonMasterCard`,
```

Comment:
Ces changements là vont plutôt dans KARL.

En fait ce qu'il faudrait faire ici c'est peut-être pas un breaking change mais juste une nouvelle feature qui serait "Add `.k-TagButton__icon` class to adds icons to tag buttons".